C7/UIElements/Popups/GameMenu.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Godot;
+
+public class GameMenu : Popup
+{
+
+    public GameMenu() {
+        alignment = BoxContainer.AlignMode.Center;
+        margins = new Margins(top: 100);
+    }
+
+    public override void _Ready()
+    {
+        base._Ready();
+
+        ImageTexture thisTexture = new ImageTexture();
+        Image image = new Image();
+        image.Create(370, 300, false, Image.Format.Rgb8);
+        image.Fill(Color.Color8(0, 0, 0, 0));
+        thisTexture.CreateFromImage(image);
+        this.Texture = thisTexture;
```

Comment:
Reading through this, I thought, "why are we doing this here?", and realized it's essentially boilerplate we have to do for every popup.  Can we toss it in another helper method, similar to AddBackground?  SetupDimensions or something of that sort?  I don't know that we can combine it with AddBackground since there are edge cases like needing 10 pixels to the right on right-aligned ones (although... shouldn't that be a margin thing?  Or done automatically for right alignment?), but we ought to be able to cut 6 lines down to 1, I'm pretty sure.

(Also could be punted for the future PR)

Reply:
It should be a margin for the 10 pixel padding as well, I didn't want to add too many changes at once, but it probably makes sense to do this in the same pr

Reply:
That looks like a good change!